lib/layout/layout.ts

```diff
@@ -25,11 +23,19 @@ function computeLayout(
   return layoutFn(getNodes(), getEdges())
 }
 
+const layoutEngines: Record<LayoutEngine, LayoutFunc> = {}
+
+export function registerLayoutEngine(
+  engine: LayoutEngine,
+  layoutFn: LayoutFunc,
+) {
+  layoutEngines[engine] = layoutFn
+}
```

Comment:
I'm pretty sure that this function needs to be exported from the `lib/index.ts` file in order to be consumed by projects when used as an external dependency.

Reply:
Indeed!